pkg/utils/dangerous_secret_names.go

```diff
@@ -0,0 +1,40 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+)
+
+var DANGEROUS_SECRET_NAMES = []string{
```

Comment:
nit: I believe you can declare this as `const`

Reply:
From [Effective Go](https://go.dev/doc/effective_go#constants)
> Constants in Go are just that—constant. They are created at compile time, even when defined as locals in functions, and can only be **numbers**, **characters** (runes), **strings** or **booleans**. Because of the compile-time restriction, the expressions that define them must be constant expressions, evaluatable by the compiler. For instance, 1<<3 is a constant expression, while math.Sin(math.Pi/4) is not because the function call to math.Sin needs to happen at run time.

Slices and maps and channels cannot be `const`.
The closest way to make it "const" is to make it private by lowercasing it and using `var dangerouSecretNames = [...]string{...}` the `[...]` notation will ensure the slice is the size of the elements it was initalized with.
e.g: 
```go
var dangerousSecretNames = [...]string{
	"PROMPT_COMMAND",
	"LD_PRELOAD",
	"LD_LIBRARY_PATH",
	"WINDIR",
	"USERPROFILE",
	"DYLD_INSERT_LIBRARIES",
}
```